components/serializer.rst

```diff
@@ -511,6 +511,26 @@ There are several types of normalizers available:
 
     Objects are normalized to a map of property names to property values.
 
+::class:`Symfony\\Component\\Serializer\\Normalizer\\JsonSerializableNormalizer`
+    This normalizer works with classes that implement `\JsonSerializable`.
```

Comment:
Please use double backticks here and below to enclose class names, method names, etc.